app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -50,13 +61,34 @@ const createSchema = (emsId, setEmsId) => ({
         isNotEmpty: true,
       },
     },
+    {
+      component: componentTypes.TEXT_FIELD,
```

Comment:
What is the purpose of hidden field?

Reply:
I used the same logic like in https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js#L33-L38.

ddf condition is only working with ddf components, so I set the fake/hidden ddf component edit that get the value 'yes' only if we are in edit mode, meaning recordId is not null (else it is null), and the conditions in the next section, the CHECKBOX, is only visible if the hidden edit component has value 'yes'